prymer/primer3/primer3_weights.py

```diff
@@ -113,6 +120,9 @@ class ProbeWeights:
     probe_tm_gt: float = 1.0
     probe_gc_lt: float = 0.5
     probe_gc_gt: float = 0.5
+    probe_self_any_th: float = 1.0
+    probe_self_end_th: float = 1.0
+    probe_hairpin_th: float = 1.0
```

Comment:
Don't these default to 0 in primer3?  E.g. [here](https://primer3.org/manual.html#PRIMER_INTERNAL_WT_SELF_ANY_TH)

Reply:
Yes -- it was an intentional decision to deviate from the primer3 defaults, I can restore them to 0.0